src/tests/unit/tests/scanner/widget-function.test.ts

```diff
@@ -95,7 +95,7 @@ describe('verify widget function data', () => {
 
     const context = {
         _data: null,
-        data: function(d: any) {
+        data: function(d): void {
```

Comment:
I'm not sure why `d: any` is not necessary here but it is used in `src/tests/unit/tests/scanner/unique-landmark-check.test.ts`(above)?

Reply:
If not necessary here because implicitly it is taking it as `any`. The case in the `unique-landmark-check.test.ts` was me being overly specific and just putting it there.